resources/energyplus/ProposedEnergy+.idd

```diff
@@ -15142,6 +15142,10 @@ AirflowNetwork:Distribution:Component:Coil,
       \key Coil:Heating:Desuperheater
       \key Coil:Heating:Electric:MultiStage
       \key Coil:Heating:Gas:MultiStage
+      \key Coil:Cooling:WaterToAirHeatPump:EquationFit
+      \key Coil:Heating:WaterToAirHeatPump:EquationFit
+      \key Coil:Cooling:WaterToAirHeatPump:VariableSpeedEquationFit
+      \key Coil:Heating:WaterToAirHeatPump:VariableSpeedEquationFit
```

Comment:
This was actually part of 25.1.0:

* https://github.com/NREL/EnergyPlus/pull/10972

Reply:
This matches the E+ IDD perfectly now.

Notice that `Coil:Cooling:DX:SingleSpeed:ThermalStorage` is **not** part of that list.

But that object does have `\reference AFNCoilNames` on it though...